db_master_deploy/bod_review.sh

```diff
@@ -54,85 +54,86 @@ sql_topics="select json_agg(row) FROM (SELECT topic, default_background, selecte
-if [[ -z "${bod_database}" ]]; then
-    echo "missing a required parameter (bod_database -b is required)" >&2
-    exit 1
-fi
-
-# check database connection
-if [[ -z $(psql -lqt -h pg-0.dev.bgdi.ch -U www-data -d ${bod_database}) ]]; then
-    echo "something went wrong when trying to connect to ${bod_database} on pg-0.dev.bgdi.ch" >&2
-    echo "please make sure that PGPASS and PGUSER Variables are set and the database name ${bod_database} is written correctly" >&2
-    exit 1
-fi
-
-# check json queries
-psql -h pg-0.dev.bgdi.ch -U www-data -d ${bod_database} -qAt -c "EXPLAIN ${sql_layer_info}" 1>/dev/null
-psql -h pg-0.dev.bgdi.ch -U www-data -d ${bod_database} -qAt -c "EXPLAIN ${sql_catalog}" 1>/dev/null
-psql -h pg-0.dev.bgdi.ch -U www-data -d ${bod_database} -qAt -c "EXPLAIN ${sql_layers_js}" 1>/dev/null
-psql -h pg-0.dev.bgdi.ch -U www-data -d ${bod_database} -qAt -c "EXPLAIN ${sql_wmtsgetcap}" 1>/dev/null
-psql -h pg-0.dev.bgdi.ch -U www-data -d ${bod_database} -qAt -c "EXPLAIN ${sql_topics}" 1>/dev/null
-
-# create temporary git folder from scratch and checkout
-mkdir -p ${git_dir}
-cd "${git_dir}"
-if [ ! -d "${git_dir}/db/.git" ]
-then
-    git clone ${git_repo}
-fi
-
-cd "${git_dir}/db"
-
-# checkout default branch
-git checkout prod 1>/dev/null
-
-git branch --merged | grep "bod_*" | xargs -r -n 1 git branch -d
-git fetch --all --prune
-
-generate_json() {
-[ -d bod_review ] || mkdir bod_review
-[ -d bod_review/re3.view_layers_js ] || mkdir bod_review/re3.view_layers_js
-
-# json export
-# re3.view_bod_layer_info_de
-psql -h pg-0.dev.bgdi.ch -U www-data -d $1 -qAt -c "${sql_layer_info}" | python -m json.tool | sed 's/ *$//' > bod_review/re3.view_bod_layer_info_de.json
-# re3.view_catalog
-psql -h pg-0.dev.bgdi.ch -U www-data -d $1 -qAt -c "${sql_catalog}" | python -m json.tool | sed 's/ *$//' > bod_review/re3.view_catalog.json
-# re3.view_layers_js
-rm bod_review/re3.view_layers_js/*.json -rf
-psql -h pg-0.dev.bgdi.ch -U www-data -d $1 -qAt -F ' ' -c "${sql_layers_js}" | while read -a Record; do
-    echo ${Record[1]} | python -m json.tool | sed 's/ *$//' > bod_review/re3.view_layers_js/${Record[0]}.json
-done
+check_access() {
+    # check env from includes.sh
+    check_env
+
+    # check for mandatory arguments source_objects and target have to be present if ArchiveMode is not set
+    if [[ -z "${bod_database}" ]]; then
+        echo "missing a required parameter (bod_database -b is required)" >&2
+        exit 1
+    fi
+
+    # check database connection
+    if [[ -z $(PSQL -lqt -U www-data -d ${bod_database}) ]]; then
+        echo "something went wrong when trying to connect to ${bod_database} on pg-0.dev.bgdi.ch" >&2
+        echo "please make sure that PGPASS and PGUSER Variables are set and the database name ${bod_database} is written correctly" >&2
+        exit 1
+    fi
+
+    # check json queries
+    PSQL -U www-data -d ${bod_database} -qAt -c "EXPLAIN ${sql_layer_info}" 1>/dev/null
```

Comment:
why do you use PSQL in Upper case ... my shell is cas sensitive and does no know PSQL, but psql?